tajo-storage/tajo-storage-common/src/main/java/org/apache/tajo/storage/StorageManager.java

```diff
@@ -538,7 +538,8 @@ public Appender getAppender(OverridableConf queryContext,
     if (scannerClass == null) {
       scannerClass = conf.getClass(
           String.format("tajo.storage.scanner-handler.%s.class", handlerName), null, Scanner.class);
-      SCANNER_HANDLER_CACHE.put(handlerName, scannerClass);
+      if (scannerClass != null)
```

Comment:
Sorry @dongjoon-hyun 

I wrote a typo for file format.
I'll test it again. :)